server/cpho/views/infobase_export.py

```diff
@@ -0,0 +1,220 @@
+    sheet_name = "indicator"
+
+    def get_column_configs(self):
+        return indicator_columns
+
+
+indicator_data_columns = [
+    ModelColumn(IndicatorDatum, "id"),
+    CustomColumn("indicator_id", lambda x: x.indicator_id),
+    # indicator columns useful to help identify??
+    CustomColumn("indicator_id", lambda x: x.indicator.name),
+    CustomColumn("indicator_id", lambda x: x.indicator.detailed_indicator),
+    CustomColumn(
+        "indicator_id", lambda x: x.indicator.sub_indicator_measurement
+    ),
+    # denormalized periods, they'll probably only use year
+    CustomColumn("year", lambda x: x.period.year),
+    CustomColumn("year - quarter", lambda x: x.period.quarter),
+    CustomColumn("year - type", lambda x: x.period.year_type),
+    # denormalized dimension attrs
+    CustomColumn("dimension_code", lambda x: x.dimension_type_id),
+    CustomColumn("dimension_en", lambda x: x.dimension_type.name_en),
+    CustomColumn("dimension_fr", lambda x: x.dimension_type.name_fr),
+    CustomColumn(
+        "is_dropdown_choice", lambda x: not (x.dimension_type.is_literal)
+    ),
+    CustomColumn(
+        "dimension_value_en",
+        lambda x: get(x, "dimension_value.name_en"),
+    ),
+    CustomColumn(
+        "dimension_value_fr",
+        lambda x: get(x, "dimension_value.name_fr"),
+    ),
+    CustomColumn(
+        "literal_dimension_val", lambda x: x.literal_dimension_val or ""
+    ),
+    # value columns
+    ChoiceColumn(IndicatorDatum, "data_quality"),
+    ChoiceColumn(IndicatorDatum, "reason_for_null"),
+    ModelColumn(IndicatorDatum, "value"),
+    ModelColumn(IndicatorDatum, "value_lower_bound"),
+    ModelColumn(IndicatorDatum, "value_upper_bound"),
+    ChoiceColumn(IndicatorDatum, "value_unit"),
+    ChoiceColumn(IndicatorDatum, "value_displayed"),
+    ModelColumn(IndicatorDatum, "single_year_timeframe"),
+    ModelColumn(IndicatorDatum, "multi_year_timeframe"),
+]
+
+
+class IndicatorDatumSheetWriter(ModelToSheetWriter):
+    sheet_name = "indicator_data"
+
+    def get_column_configs(self):
+        return indicator_data_columns
+
+
+benchmarking_columns = [
+    ModelColumn(IndicatorDatum, "id"),
+    CustomColumn("indicator_id", lambda x: x.indicator_id),
+    # indicator columns useful to help identify??
+    CustomColumn("indicator_id", lambda x: x.indicator.name),
+    CustomColumn("indicator_id", lambda x: x.indicator.detailed_indicator),
+    CustomColumn(
+        "indicator_id", lambda x: x.indicator.sub_indicator_measurement
+    ),
+    ChoiceColumn(Benchmarking, "unit"),
+    ModelColumn(Benchmarking, "oecd_country"),
+    ModelColumn(Benchmarking, "value"),
+    ModelColumn(Benchmarking, "year"),
+    ChoiceColumn(Benchmarking, "comparison_to_oecd_avg"),
+    ChoiceColumn(Benchmarking, "labels"),
+]
+
+
+class BenchmarkingSheetWriter(ModelToSheetWriter):
+    def get_column_configs(self):
+        return benchmarking_columns
+
+
+trend_columns = [
```

Comment:
We might need to add indicator metadata columns here especially the indicator_id, 

We could also add the following (indicator.name,  indicator.detailed_indicator, x.indicator.sub_indicator_measurement) for better readability of the file